Formula/opencbm.rb

```diff
@@ -1,10 +1,15 @@
 class Opencbm < Formula
   desc "Provides access to various floppy drive formats"
-  homepage "https://spiro.trikaliotis.net/opencbm-alpha"
+  homepage "https://spiro.trikaliotis.net/opencbm"
   url "https://spiro.trikaliotis.net/Download/opencbm-0.4.99.99/opencbm-0.4.99.99.tar.bz2"
   sha256 "b1e4cd73c8459acd48c5e8536d47439bafea51f136f43fde5a4d6a5f7dbaf6c6"
   head "https://git.code.sf.net/p/opencbm/code.git"
```

Comment:
```suggestion
  license "GPL-2.0-only"
  head "https://git.code.sf.net/p/opencbm/code.git"
```

Reply:
I was just about to add this to the PR before you commented (referencing the `README` and `COPYING` file on SourceForge), so I appreciate the confirmation.